Automattic/wp-calypso
# Stats: Moving UTM to the module grid

Related to #87569

## Proposed Changes

* moving the new UTM module to the modules grid according to the design
* updating other module order when UTM module is visible
* updating button colour to match Jetpack
* updating `learn more` button for the module

## Testing Instructions



* open the live branch
* apply `stats/utm-module` feature flag
* verify that the new module is below the country module
* check different breakpoints and verify that they modules align to the edges

![SCR-20240229-lpgu](https://github.com/Automattic/wp-calypso/assets/112354940/fc6033f2-721b-4283-9f33-2809abfbd28e)


## Pre-merge Checklist
